README.md

```diff
@@ -1,6 +1,33 @@
 # OpenSAFELY Command Line Interface
 
-_User-facing documentation to go here_
+This tool is part of the [OpenSAFELY](https://www.opensafely.org)
+project.
+
+To use it you will need:
+ * Python version 3.7 or above
+ * git
+ * Docker
+
+Once these are installed you can install the tool itself with:
+```
+pip install opensafely
+```
+
+At present there is one command, which can be used to run actions in an
```

Comment:
I think this is begging to become stale! The documentation *is* the argparse help.
 Better to say "run `opensafely --help` to see permitted arguments"?


https://github.com/DataDog/mkdocs-click

Reply:
:+1: quite right